packages/itwinui-css/src/file-upload/file-upload.scss

```diff
@@ -74,3 +75,56 @@
     }
   }
 }
+
+@mixin iui-file-card {
+  width: 100%;
+  height: 100%;
+  line-height: normal;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+
+  > svg,
+  .iui-file-card-icon {
+    @include iui-icon-style(xl);
+    @include iui-icon-color;
+    margin-top: var(--iui-size-s);
+    margin-bottom: var(--iui-size-s);
+    margin-left: var(--iui-size-xs);
+  }
+
+  .iui-browse-input {
+    width: 0.1px;
+    height: 0.1px;
+    opacity: 0;
+    overflow: hidden;
+    position: absolute;
+  }
+
+  .iui-file-card-action {
```

Comment:
most of these classes should be separate mixins instead of nesting. and then the classes can be defined in `classes.scss`

Reply:
These classes have been moved to separate mixins